src/Raven.Client/Documents/Changes/DatabaseChanges.cs

```diff
@@ -47,14 +47,21 @@ public DatabaseChanges(RequestExecutor requestExecutor, string databaseName, Act
            
             _tcs = new TaskCompletionSource<IDatabaseChanges>(TaskCreationOptions.RunContinuationsAsynchronously);
             _cts = new CancellationTokenSource();
-            _client = new ClientWebSocket();
+            _client = CreateClientWebSocket(_requestExecutor);
 
             _onDispose = onDispose;
             ConnectionStatusChanged += OnConnectionStatusChanged;
 
             _task = DoWork();   
         }
 
+        public static ClientWebSocket CreateClientWebSocket(RequestExecutor requestExecutor)
+        {
+            var clientWebSocket = new ClientWebSocket();
+            clientWebSocket.Options.ClientCertificates.Add(requestExecutor.Certificate);
```

Comment:
shouldn't we do if `requestExecutoer.Certififate != null` ?

Reply:
Will fix this.